src/foam/nanos/crunch/Capability.js

```diff
@@ -223,6 +223,14 @@ foam.CLASS({
       javaFactory: `
         return foam.nanos.crunch.AssociatedEntity.USER;
       `
+    },
+    {
+      name: 'wizardConfig',
+      class: 'FObjectProperty',
+      of: 'foam.u2.wizard.StepWizardConfig',
+      documentation: `
+        Allow this capability to configure the wizard when it is the root capability.
```

Comment:
Does this not need a default value or factory? I was also thinking this would be a generic class reference string, which could then be looked up and utilized by the stepWizardletController.

Reply:
I can be set as `{ class: 'foam.u2.wizard.RefreshingStepWizardConfig' }` which also allows setting other properties